app/controllers/users_controller.rb

```diff
@@ -7,6 +7,9 @@ def new
 
   def create
     @user = User.create(user_params)
+    if @user
+      session[:user_id] = @user.id
+    end
```

Comment:
This does not belong in this PR (if this PR is about email validation).

Also, since this is logging the user in, then a suggestion to pull out this into its own method that can be called here and in the sessions controller.

Reply:
Sorry, didn't really name this PR well at all, it was meant to be a general validation/confirmation PR.
Done in sha: 1159cb8